Remove app deployment guides from developer docs (#1307)
mattermost/mattermost-developer-documentation: site/content/integrate/apps/quickstart/quick-start-serverless/_index.md

- Contains a `manifest.json`, declares itself an HTTP application that uses a bot account, uses webhooks, and attaches UI elements to locations in the Mattermost interface.
- Attaches `send` function to a `/hello-serverless` command, accepting an optional `message` argument.

> This App will focus on the functionality of the actual serverless App in this example. For more information about how to package a Mattermost App for serverless deployments, see the {{< newtabref href="https://docs.mattermost.com/guides/package-for-aws.html" title="deploy a Mattermost App to AWS" >}} documentation, or {{< newtabref href="https://docs.mattermost.com/guides/deploy-openfaas.html" title="deploy a Mattermost App to OpenFaaS" >}} documentation.

Reviewer: ```suggestion
This App will focus on the functionality of the actual serverless App in this example. For more information about how to package a Mattermost App for serverless deployments, see the {{< newtabref href="https://docs.mattermost.com/guides/deploy-aws.html" title="deploy a Mattermost App to AWS" >}} documentation, or {{< newtabref href="https://docs.mattermost.com/guides/deploy-openfaas.html" title="deploy a Mattermost App to OpenFaaS" >}} documentation.
```

> This App will focus on the functionality of the actual serverless App in this example. For more information about [how to package a Mattermost App for serverless deployments]("https://docs.mattermost.com/guides/package-for-aws.html"), [how to deploy a Mattermost App to AWS]("https://docs.mattermost.com/guides/deploy-aws.html"), or [how to deploy a Mattermost App to OpenFaaS]("https://docs.mattermost.com/guides/deploy-openfaas.html") please refer to the corresponding guides.

Reviewer: ```suggestion
This App will focus on the functionality of the actual serverless App in this example. For more information about how to package a Mattermost App for serverless deployments, see the {{< newtabref href="https://docs.mattermost.com/guides/package-for-aws.html" title="deploy a Mattermost App to AWS" >}} documentation, or {{< newtabref href="https://docs.mattermost.com/guides/deploy-openfaas.html" title="deploy a Mattermost App to OpenFaaS" >}} documentation.
```

docker compose up
```

> The `docker-compose.yml` uses the [HTTP deployment]("https://docs.mattermost.com/guides/deploy-http.html") of the serverless App for the sake of trying it out locally. You'll see Docker install the Go modules and then the App will come online and print the following message:

Reviewer: ```suggestion
The `docker-compose.yml` uses the {{< newtabref href="https://docs.mattermost.com/guides/deploy-http.html" title="HTTP deployment" >}} of the serverless App for the sake of trying it out locally. You'll see Docker install the Go modules and then the App will come online and print the following message:
```

## Explore serverless deployments

> Now that you've examined the App in HTTP mode, you can next look at deploying them to your desired serverless plaform. Check out the guides on [how to deploy a Mattermost App to AWS]("https://docs.mattermost.com/guides/deploy-aws.html") or [how to deploy a Mattermost App to OpenFaaS]( "https://docs.mattermost.com/guides/deploy-openfaas.html"), depending on which platform you're targeting.

Reviewer: ```suggestion
Now that you've examined the App in HTTP mode, you can next look at deploying them to your desired serverless plaform. Check out the guides on {{< newtabref href="https://docs.mattermost.com/guides/deploy-aws.html" title="how to deploy a Mattermost App to AWS" >}} or {{< newtabref href="https://docs.mattermost.com/guides/deploy-openfaas.html" title="how to deploy a Mattermost App to OpenFaaS" >}}, depending on which platform you're targeting.
```
